### PR TITLE
Better dependencies Seq

### PR DESCRIPTION
**Pull request by s5bug**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -4,9 +4,10 @@ lazy val root = (project in file("."))
     organization := "github.discordscala",
     scalaVersion := "2.12.4",
     version      := "0.1.0-SNAPSHOT",
-    libraryDependencies ++= Seq("net.liftweb" %% "lift-json" % "3.1.1",
+    libraryDependencies ++= Seq(
+      "net.liftweb" %% "lift-json" % "3.1.1",
```
Comment (justgerd) [inline build.sbt:8]: Here formatting seems a little bit messed up relative to the other entries

File: build.sbt [hunk 2]
```diff
@@ -4,9 +4,10 @@ lazy val root = (project in file("."))
...
     	"com.typesafe.akka" %% "akka-http" % "10.0.11",
     	"com.typesafe.akka" %% "akka-actor" % "2.5.8",
-    	"com.lihaoyi" %% "fastparse" % "1.0.0"
+    	"com.lihaoyi" %% "fastparse" % "1.0.0",
```
Comment (justgerd) [inline build.sbt:11]: why that additional colon?

**Review by justgerd — APPROVED** [review]
Fix that spacing there.

File: build.sbt (hunk 1 above)
Comment (s5bug) [reply build.sbt:8]: You usually
```
Seq(
  element,
  element,
)
```
in a multiline seq

**Pull request merged**